usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -119,17 +116,14 @@ def signal_handler(signal, frame):
                     .format(submission_id, award_financial_frame.shape[0]))
         logger.info('Loading File C data')
         start_time = datetime.now()
-        awards_touched = load_file_c(submission_attributes, db_cursor, award_financial_frame)
+        load_file_c(submission_attributes, db_cursor, award_financial_frame)
         logger.info('Finished loading File C data, took {}'.format(datetime.now() - start_time))
 
+        # update C to D linkage using SQL scripts
+        call_command('update_file_c_linkages')
```

Comment:
@nmonga91 what is the usual runtime of this sql? 
Would it be simple to create a version of this that takes in a submission ID and somehow narrows the awards that need to be linked, rather than attempting to fill _all_ gapped links? Or is this pretty efficient as-is, in that it targets just those `financial_accounts_by_award` records with a `null` `award_id`?

Reply:
It targets just those `financial_accounts_by_award` records with a `null` `award_id`. It'll still operate on _all_ unlinked records so it's not the most efficient method here. I think it's best to move to a submission specific SQL set for this.